### ISSUE TITLE
Feature requests

### ISSUE THREAD

**Issue by wpsumo** [elightup/falcon#21]
Nice you have a plugin for this as well.

Can I request a few features as I don't like any of the Admin and Site Enhancements free/pro and  extended free/pro.

Missing important features. Just not sure how much you prioritize and have time for this plugin if its worth adding my feature requests :) 

Thanks in advance!

**Comment by rilwis** [comment]
Yes, we're happy to accept feature requests here, or better if you [send us an email](https://elightup.com/contact/).
